src/plugins/loan-use-filter.common.js

```diff
@@ -0,0 +1,69 @@
+const numeral = require('numeral');
```

Comment:
This file is a duplicate of src/plugins/loan-use-filter.js but with module.exports to make node 12 happy.

Reply:
You could also just change loan-use-filter.js. It should still work when using `module.exports` instead of `export`.